srv/salt/_modules/wait.py

```diff
@@ -74,30 +79,32 @@ def _check_status(self, current):
         """
         Return the "correct" matching status
         """
+        # pylint: disable=no-else-return
         if self.settings['negate']:
             log.debug("status != {}".format(self.settings['status']))
-            return (current != self.settings['status'])
+            return current != self.settings['status']
         else:
             log.debug("status == {}".format(self.settings['status']))
-            return (current == self.settings['status'])
+            return current == self.settings['status']
 
 
+# pylint: disable=too-few-public-methods
 class FsStatusCheck(HealthCheck):
     """
     Check the fsmap status of the ceph status output. Wait till all active MDS's
     daemons have reached up:active status
     """
 
-    def __init__(self, **kwargs):
-        super(FsStatusCheck, self).__init__(**kwargs)
-
     def wait_for_healthy_mds(self):
         """
         Poll until all active MDS' are up:active
         """
-        cmd = json.dumps({"prefix":"status", "format":"json" })
+        cmd = json.dumps({"prefix": "status", "format": "json"})
 
         def success(status):
+            """
+            success function to passed to _check_status
+            """
```

Comment:
I believe you mean: "Success function to be passed to _check_status". Also capitalization.

Reply:
Absolutely! Thanks.